integration/test_detectionlists.py

```diff
@@ -1,125 +1,120 @@
 
-def test_departing_employee_add(connection):
-    response = connection.detectionlists.departing_employee.add(new_user)
-    assert response.status_code == 200
+    def test_refresh_user_scim_attributes(self, connection, new_user):
+        response = connection.detectionlists.refresh_user_scim_attributes(
+            new_user["userUid"]
+        )
+        assert response.status_code == 200
 
+    def test_departing_employee_get_page(self, connection):
+        response = connection.detectionlists.departing_employee.get_page(1)
+        assert response.status_code == 200
 
-def test_departing_employee_update_departure_date(connection):
-    response = connection.detectionlists.departing_employee.update_departure_date(
-        user_uid, user_departure_date
-    )
-    assert response.status_code == 200
+    def test_departing_employee_get_all(self, connection):
+        response_gen = connection.detectionlists.departing_employee.get_all()
+        for response in response_gen:
+            assert response.status_code == 200
+            break
+
+    def test_high_risk_employee_get_all(self, connection):
+        response_gen = connection.detectionlists.high_risk_employee.get_all()
+        for response in response_gen:
+            assert response.status_code == 200
+            break
+
+    def test_add_user_cloud_alias(self, connection, new_user):
+        response = connection.detectionlists.add_user_cloud_alias(
+            new_user["userUid"], alias_user
+        )
+        assert response.status_code == 200
 
+    def test_remove_user_cloud_alias(self, connection, new_user):
+        response = connection.detectionlists.remove_user_cloud_alias(
+            new_user["userUid"], alias_user
+        )
+        assert response.status_code == 200
 
-def test_departing_employee_remove(connection):
-    response = connection.detectionlists.departing_employee.remove(new_user)
-    assert response.status_code == 200
+    def test_departing_employee_add(self, connection, new_user):
+        response = connection.detectionlists.departing_employee.add(new_user["userUid"])
+        assert response.status_code == 200
 
+    def test_departing_employee_get(self, connection, new_user):
+        response = connection.detectionlists.departing_employee.get(new_user["userUid"])
+        assert response.status_code == 200
 
-def test_departing_employee_set_alerts_enabled(connection):
-    response = connection.detectionlists.departing_employee.set_alerts_enabled()
-    assert response.status_code == 200
+    def test_departing_employee_update_departure_date(self, connection, new_user):
+        response = connection.detectionlists.departing_employee.update_departure_date(
+            new_user["userUid"], user_departure_date
+        )
+        assert response.status_code == 200
 
+    def test_add_user_risk_tags(self, connection, new_user):
+        response = connection.detectionlists.add_user_risk_tags(
+            new_user["userUid"], "Flight_Risk"
```

Comment:
Is `Flight_Risk` a system specific data, if so then need to make it configurable, please confirm.

Reply:
We have the enum values defined in `py42.clients.detectionlists.RiskTags`, I think it's better to use the enum values.

I don't think it needs to be configurable, but this method accepts `str or list of str`, so we can give it multiple in one go, if you want to try it out.

Reply:
Changed.